PATCH_NOTES.txt

```diff
@@ -1,117 +1,4 @@
-# Starlight River v0.2.0: The Power Of Friendship is now live!
```

Comment:
Do we want to totally nuke this file? Part of me thinks it may be better to have newer patches simply exist above older ones

Reply:
it'll get very very long over time that way. another alternative is to put old ones into a archived patchnotes folder or something